src/Config.php

```diff
@@ -54,93 +55,75 @@ class Config
      * @param bool $compression cachovat WDSL?
      * @throws InvalidArgumentException
      */
-    public function __construct($appId, $isTestMode = false, $cache = true, $compression = true)
-    {
+    public function __construct(
+      string $appId,
+      bool $isTestMode = self::TESTMODE_DISABLED,
+      bool $cache = self::CACHE_ENABLED,
+      bool $compression = self::COMPRESSION_ENABLED
+    ) {
         if (empty($appId)) {
-            throw new InvalidArgumentException("AppId cannot be empty.");
+            throw new InvalidArgumentException('AppId cannot be empty.');
         }
         $this->appId = $appId;
         $this->setTestMode($isTestMode);
         $this->setCache($cache);
         $this->setCompression($compression);
     }
 
-    /**
-     * @return string
-     */
-    public function getAppId()
+    public function getAppId(): string
     {
         return $this->appId;
     }
 
-    /**
-     * @return bool
-     */
-    public function isTestMode()
+    public function isTestMode(): bool
     {
         return $this->testMode;
     }
 
-    /**
-     * @param bool $isTestMode
-     * @return self
-     */
-    public function setTestMode($isTestMode = true)
+    public function setTestMode(bool $isTestMode = true): self
     {
-        $this->testMode = (bool) $isTestMode;
+        $this->testMode = $isTestMode;
         return $this;
     }
 
     /**
      * Zjistí, jestli je WSDL cachované
-     *
-     * @return bool
      */
-    public function getCache()
+    public function getCache(): bool
```

Comment:
out of scope, ale nestálo by v budoucnu použít nějaký popisnější název? `isCacheEnabled()` apod

Reply:
Ano, mas pravdu. Tohle a par dalsich zmen chystam jako dalsi pull request.